backend/src/main/java/com/google/backend/servlets/DataServlet.java

```diff
@@ -25,10 +25,9 @@
 */
 @WebServlet("/api/v1/test-servlet")
 public class DataServlet extends HttpServlet {
-
- @Override
- public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-   response.setContentType("text/html;");
-   response.getWriter().println("Hello world from data servlet!");
- }
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Even if the functionality is very basic, good to at least include the skeleton code for the `DataServletTest` and add that file to the corresponding directory as well.